jbpm-services/jbpm-executor/src/test/resources/BPMN2-AsyncSubProcess.bpmn2

```diff
@@ -0,0 +1,119 @@
+<?xml version="1.0" encoding="UTF-8"?> 
+<definitions id="Definition"
+             targetNamespace="http://www.example.org/MinimalExample"
+             typeLanguage="http://www.java.com/javaTypes"
+             expressionLanguage="http://www.mvel.org/2.0"
+             xmlns="http://www.omg.org/spec/BPMN/20100524/MODEL"
+             xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+             xsi:schemaLocation="http://www.omg.org/spec/BPMN/20100524/MODEL BPMN20.xsd"
+             xmlns:bpmndi="http://www.omg.org/spec/BPMN/20100524/DI"
+             xmlns:dc="http://www.omg.org/spec/DD/20100524/DC"
+             xmlns:di="http://www.omg.org/spec/DD/20100524/DI"
+             xmlns:tns="http://www.jboss.org/drools">
+
+  <itemDefinition id="_2-xItem" structureRef="String" />
+
+  <process processType="Private" isExecutable="true" id="AsyncSubProcess" name="Minimal SubProcess" >
+
+    <!-- nodes -->
+    <startEvent id="_1" name="StartProcess" />
+    <subProcess id="_2" name="Hello"  >
+      <extensionElements>
+        <tns:metaData name="customAsync">
+          <tns:metaValue>true</tns:metaValue>
+        </tns:metaData>
+      </extensionElements>    <!-- variables -->
+    <property id="x" itemSubjectRef="_2-xItem"/>
+    <!-- nodes -->
+    <startEvent id="_2-1" name="StartSubProcess" />
+    <scriptTask id="_2-2" name="Hello1" >
+      <script>System.out.println(Thread.currentThread().getName() + " x = " + x);</script>
+    </scriptTask>
+    <scriptTask id="_2-3" name="Hello2" >
+      <script>kcontext.setVariable("x", "Hello");</script>
+    </scriptTask>
+    <scriptTask id="_2-4" name="Hello3" >
+      <script>System.out.println(Thread.currentThread().getName() + " x = " + x);</script>
+    </scriptTask>
+    <endEvent id="_2-5" name="EndSubProcess" />
+    <!-- connections -->
+    <sequenceFlow id="_2-1-_2-2" sourceRef="_2-1" targetRef="_2-2" />
+    <sequenceFlow id="_2-2-_2-3" sourceRef="_2-2" targetRef="_2-3" />
+    <sequenceFlow id="_2-3-_2-4" sourceRef="_2-3" targetRef="_2-4" />
+    <sequenceFlow id="_2-4-_2-5" sourceRef="_2-4" targetRef="_2-5" />
+    </subProcess>
```

Comment:
Well I think so, I just copied the definition and added <extensionElements> with customAsync metadata. I think I will format it properly in IDE.